app.py

```diff
@@ -4,7 +4,7 @@
 from dash.exceptions import PreventUpdate
 
 app = dash.Dash(external_stylesheets=[dbc.themes.BOOTSTRAP])
-
+server = app.server
```

Comment:
is this needed? i don't see it being used :o

Reply:
ok nevermind i found [an explaination](https://towardsdatascience.com/deploying-your-dash-app-to-heroku-the-magical-guide-39bd6a0c586c)

Reply:
@timothytancy ?